control-plane/pkg/reconciler/broker/broker_test.go

```diff
@@ -136,24 +144,19 @@ func brokerReconciliation(t *testing.T, format string, configs Configs) {
 		{
 			Name: "Reconciled normal - with DLS",
 			Objects: []runtime.Object{
-				NewDeletedBroker(
+				NewBroker(
```

Comment:
This test was actually wrong because here we're testing `ReconcileKind` and not `FinalizeKind`, so I fixed it.